src/Microsoft.Extensions.Http/Logging/EventIds.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Extensions.Http.Logging
+{
+    internal static class EventIds
+    {
+        // Logging done by LoggingScopeHttpMessageHandler - this surrounds the whole pipeline
+        public static readonly EventId RequestPipelineStart = new EventId(100, "RequestPipelineStart");
```

Comment:
Ugh. I dislike artificially spacing out Event Ids. Just leads to weird patterns and trying to assign meaning to things that are really just arbitrary integers identifying an event. In this case, it's not likely we'll overrun these regions but it still just seems unnecessary. The names are descriptive enough and if we document it, we'll just have a list of IDs and what they map to and I don't think users will care if they aren't "nicely ordered"

Reply:
Ok, I'll just not do this.